### PR TITLE
Fix Typo - English

### PR DESCRIPTION
**Pull request by parapenT1sta**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1405,7 +1405,7 @@
     <string name="donate_your_data_to_science">Donate your data to science</string>
     <string name="open_humans_short">OH</string>
     <string name="you_have_been_signed_out_of_open_humans">You have been signed out of Open Humans</string>
-    <string name="click_here_to_sign_in_again_if_this_wasnt_on_purpose">Click here to sign in a again if this wasn\'t on purpose.</string>
```
Comment (rICTx-T1D) [inline app/src/main/res/values/strings.xml:1408]: The blacksplash was needed to escape apostrophe.

Comment (parapenT1sta) [reply app/src/main/res/values/strings.xml:1408]: @rICTx-T1D Sorry. But the "a". Shouldn't be before "again". Correct?

Comment (parapenT1sta) [reply app/src/main/res/values/strings.xml:1408]: Just fixed it. Is it ok now?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1405,7 +1405,7 @@
     <string name="donate_your_data_to_science">Donate your data to science</string>
     <string name="open_humans_short">OH</string>
     <string name="you_have_been_signed_out_of_open_humans">You have been signed out of Open Humans</string>
-    <string name="click_here_to_sign_in_again_if_this_wasnt_on_purpose">Click here to sign in again if this wasn't on purpose.</string>
+    <string name="click_here_to_sign_in_again_if_this_wasnt_on_purpose">Click here to sign in again if this wasn\'t on purpose.</string>
```
Comment (rICTx-T1D) [inline app/src/main/res/values/strings.xml:1408]: Look like correctly so.

**Review by rICTx-T1D — APPROVED** [review]

**Pull request merged**